first.css

```diff
@@ -1,3 +1,3 @@
-.header{
-    color: red;
-}
```

Comment:
- [ ] Every Pull Request should have a descriptive **_title_** telling users what changes are inside. Also, every Pull Request should have a summary describing introduced changes in more detail.

**Your pull request**
![image](https://user-images.githubusercontent.com/84156714/220326540-d0ef2421-986a-4974-b9ee-03829f8b8d4a.png)

**Good example PR**

![image](https://user-images.githubusercontent.com/84156714/220326681-c907fbd4-4713-4995-843b-942fedcffb41.png)